README.md

```diff
@@ -34,11 +34,8 @@ Daily Builds
 |---------|:----------:|:----------:|:----------:|
 |   **Windows (x64)**    |   [![][win-x64-badge-master]][win-x64-version-master]<br>[Installer][win-x64-installer-master]<br>[zip][win-x64-zip-master]<br>[Symbols (zip)][win-x64-symbols-zip-master]   |   [![][win-x64-badge-1.1.X]][win-x64-version-1.1.X]<br>[Installer][win-x64-installer-1.1.X]<br>[zip][win-x64-zip-1.1.X]   |   [![][win-x64-badge-preview]][win-x64-version-preview]<br>[Installer][win-x64-installer-preview]<br>[zip][win-x64-zip-preview]   |
 |   **Windows (x86)**    |   [![][win-x86-badge-master]][win-x86-version-master]<br>[Installer][win-x86-installer-master]<br>[zip][win-x86-zip-master]<br>[Symbols (zip)][win-x86-symbols-zip-master]   |   [![][win-x86-badge-1.1.X]][win-x86-version-1.1.X]<br>[Installer][win-x86-installer-1.1.X]<br>[zip][win-x86-zip-1.1.X]   |   [![][win-x86-badge-preview]][win-x86-version-preview]<br>[Installer][win-x86-installer-preview]<br>[zip][win-x86-zip-preview]   |
-|   **Windows (arm32)**  |   [![][win-arm-badge-master]][win-arm-version-master]<br>[zip][win-arm-zip-master]<br>[Symbols (zip)][win-arm-symbols-zip-master]   |   N/A   |   N/A   |
```

Comment:
Why are we removing these?  I think the intention is to get these back up and running.